spec/models/task_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+
+RSpec.describe Task, type: :model do
+  before do
+    FactoryBot.create(:user)
+  end
```

Comment:
1行のbeforeは下記のように記載できます。
```suggestion
  before { FactoryBot.create(:user) }
```